sycl/test-e2e/Plugin/interop-opencl-interop-task-mem.cpp

```diff
@@ -1,4 +1,5 @@
 // REQUIRES: opencl, opencl_icd
+// REQUIRES: aspect-ext_intel_legacy_image
```

Comment:
Good find! I think it should be possible to have it in the same line as the requirements above, but I don't mind separating the aspect requirements line when possible.